tfmesos/scheduler.py

```diff
@@ -324,16 +327,26 @@ def readable(fd):
                 self, framework, self.master, use_addict=True
             )
             self.driver.start()
-            while any((not task.initalized for task in self.tasks)):
+            task_start_count = 0
+            while any((not task.initalized
+                       for id, task in self.tasks.iteritems())):
                 if readable(lfd):
                     c, _ = lfd.accept()
                     if readable(c):
                         mesos_task_id, addr = recv(c)
-                        assert isinstance(mesos_task_id, int)
                         task = self.tasks[mesos_task_id]
                         task.addr = addr
                         task.connection = c
                         task.initalized = True
+                        task_start_count += 1
+                        logger.info('Task %s with mesos_task_id %s has '
```

Comment:
Can we keep this kind of logs `DEBUG` instead of `INFO` ? I am afraid this is a bit too noisy for most users?